shared/js/background/abp-lists.es6.js

```diff
@@ -178,11 +178,14 @@ chrome.runtime.onInstalled.addListener(function(details) {
             chrome.tabs.query({currentWindow: true, active: true}, function(tabs) { 
                 const domain = (tabs && tabs[0]) ? tabs[0].url : ''
                 const regExpPostInstall = new RegExp('duckduckgo\.com\/app')
-                if ((!settings.getSetting('hasSeenPostInstall')) && (!domain.match(regExpPostInstall))) {
-                    settings.updateSetting('hasSeenPostInstall', true)
-                    chrome.tabs.create({
-                        url: 'https://duckduckgo.com/app?post=1'
-                    })
+                const regExpSoftwarePage = new RegExp('duckduckgo\.com\/software')
+                if ((!settings.getSetting('hasSeenPostInstall'))
+                    && (!domain.match(regExpPostInstall))
+                    && (!domain.match(regExpSoftwarePage))) {
+                        settings.updateSetting('hasSeenPostInstall', true)
+                        chrome.tabs.create({
+                            url: 'https://duckduckgo.com/app?post=1'
+                        })
```

Comment:
This is exactly the sort of thing we should be adding unit tests for. Maybe we can abstract away `shouldShowPostInstall` to a separate method somewhere and add tests for that.

Adding new tests would be easier after https://github.com/duckduckgo/duckduckgo-privacy-extension/pull/184 gets merged, but we may still need to restructure the code a bit to get that going.